sdk/src/main/java/com/mapbox/maps/renderer/MapboxRenderThread.kt

```diff
@@ -46,17 +49,23 @@ internal class MapboxRenderThread : Choreographer.FrameCallback {
   private var width: Int = 0
   private var height: Int = 0
 
+  private val widgetRenderer : MapboxWidgetRenderer
+  private lateinit var glDrawWidgets: GLDrawTexture
```

Comment:
very strange plural naming, could you please align it

Reply:
Well, it draw widgets, what do you think is the right name for it?) Seems ok to me